cfgov/jinja2/v1/owning-a-home/_templates/form-explainer.html

```diff
@@ -0,0 +1,235 @@
+                        {
+                         'category':   'String containing the name of the category.'
+                         'definition': 'This definition can contain HTML.',
+                         'highlight':  'String containing a url to an image.'
+                         'id':         'a-unique-html-id.',
+                         'term':       'Term name.'
+                        }
+
+   index:              The term's position within the array.
+
+   page_num:           The page that this term belongs to.
+
+   ========================================================================== #}
+
+{% macro term( item, index, page_num ) %}
+{% if item.definition == '' %}
+<div class="o-expandable o-expandable__padded
+            o-expandable__form-explainer
+            o-expandable__form-explainer-{{ item.category }}"
+     id="{{ 'page-' + page_num |string + '-' + item.category + '-' + item.id }}"
+     tabindex="0">
+    <span class="o-expandable_header">
+        <span class="o-expandable_label">
+            <span class="o-expandable_category o-expandable_category__{{ item.category }}">
+                <span class="u-visually-hidden">{{ item.category }}</span>
+            </span>
+            {{ item.term }}
+        </span>
+    </span>
+</div>
+{% else %}
+  <div class="o-expandable o-expandable__padded
+              o-expandable__form-explainer
+              o-expandable__form-explainer-{{ item.category }}"
+       id="{{ 'page-' + page_num|string + '-' + item.category + '-' + item.id }}">
+    <button class="o-expandable_header o-expandable_target" tabindex="0">
+        <span class="o-expandable_header-left o-expandable_label">
+            <span class="o-expandable_category o-expandable_category__{{ item.category }}">
+                <span class="u-visually-hidden">{{ item.category }}</span>
+            </span>
+            {{ item.term }}
+        </span>
+        <span class="o-expandable_header-right o-expandable_link">
+            <span class="o-expandable_cue-open">
+                <span class="u-visually-hidden">Show</span>
+                <span class="cf-icon cf-icon-plus"></span>
+            </span>
+            <span class="o-expandable_cue-close">
+                <span class="u-visually-hidden">Hide</span>
+                <span class="cf-icon cf-icon-minus"></span>
+            </span>
+        </span>
+    </button>
+    <div class="o-expandable_content u-stick" tabindex="0">
+        {{ item.definition|safe }}
+        {% if item.highlight -%}
+            <img class="u-stick_bottom" src="{{ item.highlight }}" alt="">
+        {%- endif %}
+    </div>
+</div>
+{% endif %}
+{% endmacro %}
+
+
+{# ==========================================================================
+
+   render_pagination( pages )
+
+   ==========================================================================
+
+   Description:
+
+   Render pagination.
+
+   pages: Pages in form
```

Comment:
The structure of `pages` could be explained here.

Reply:
Fixed.